src/clojure/nrepl/cmdline.clj

```diff
@@ -271,8 +272,10 @@
             handler (or handler (build-handler middleware))
             transport (some->> (:transport options) (require-and-resolve :transport))
             greeting-fn (if (= transport #'transport/tty) #'transport/tty-greeting)
+            verbose (:verbose options)
```

Comment:
I guess we should also check if this was set in the config file here.

Reply:
Gonna code it o/

Reply:
It already works because `options` is already merged, but I've added it to the docs o/

Reply:
Ah, excellent!